bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,230 @@ class StylintBear:
+            ``False``, expect no brackets when declaring a selector.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param colons:
+            When ``True``, expect ``:`` when declaring a property. When
+            ``False``, expect no ``:`` when declaring a property.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param colors:
+            When ``True``, enforce variables when defining hex values.
+            Supported values are ``True`` and ``None``.
+        :param spaces_after_commas:
+            Enforce or disallow spaces after commas.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param spaces_after_comments:
+            Enforce or disallow spaces after line comments.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param trailing_whitespace:
+            If ``None``, ignores trailing white space. If ``False``, trailing
+            white space will throw a warning.
+        :param css_literal:
+            By default Stylint ignores ``@css`` blocks. If set to true however,
+            it will throw a warning if ``@css`` is used.
+            Supported values are ``False`` and ``None``.
+        :param max_selector_depth:
+            Set the max selector depth. If set to 4, max selector depth will
+            be 4 indents. Pseudo selectors like ``&:first-child`` or
+            ``&:hover`` won't count towards the limit.
+        :param check_duplicates:
+            Checks if selector or property duplicated unnecessarily.
+        :param efficient_properties:
+            Check for places where properties can be written more efficiently.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+            Example if set to ``@extends``: prefer
+            ``@extends $some-var`` over ``@extend $some-var``
+            Example if set to ``@extend:`` prefer
+            ``@extend $some-var`` over ``@extends $some-var``
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``0``.
+            Example if ``2``: prefer
+            ``/s/smargin: 0`` over ``/s/s/smargin: 0``
+        :param leading_zero:
+            When ``True``, prefer leading zeroes on decimal points. When
+            ``False``, disallow leading zeroes on decimal points.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param max_errors:
+            Set 'max' number of Errors. Use ``0`` to disable this feature.
+        :param max_warnings:
+            Set 'max' number of Warnings. Use ``0`` to disable this feature.
```

Comment:
I think you should rephrase

Use 0 to disable this feature.

to

If 0, all errors will be shown without a limit.

or similar^^